.github/workflows/pr-open.yml

```diff
@@ -374,6 +375,24 @@ jobs:
           oc rollout status dc/${{ env.NAME }}-${{ env.ZONE }}-backend -w
           oc rollout status dc/${{ env.NAME }}-${{ env.ZONE }}-frontend -w
 
+      - name: Deployment Verification
+        env:
+          LINKS: (
+            'https://${{ env.NAME }}-${{ github.event.number }}-backend.${{ env.DOMAIN }}'
+            'https://${{ env.NAME }}-${{ github.event.number }}-frontend.${{ env.DOMAIN }}'
```

Comment:
Hi Derek, I am sure your change works, but just wondering why I am not seeing "env.NAME" in the "env:" at line 335? or maybe they it is from other place defined for 'env'?

Reply:
@ianliuwk1019 Good ask!  It's on line 8, set for the entire workflow.  379-382 are for this step only.  :)